css/zoom.css

```diff
@@ -35,3 +35,11 @@ img.zoom-img {
   filter: "alpha(opacity=100)";
   opacity: 1;
 }
+.zoom-overlay-open {
```

Comment:
nice work man! But could you move this definition up and combine it with `img.zoom-img` thanks!